src/Code.js

```diff
@@ -7,22 +7,74 @@ const URL_BASE = '.multibaas.com/api/v0/';
 const HTTP_GET = 'GET';
 const HTTP_POST = 'POST';
 
-/**
- * Add menu
- */
-function onOpen() {
+// Property keys for deployment ID and API key.
+const PROP_MB_DEPLOYMENT_ID = 'mbDeploymentId';
+const PROP_MB_API_KEY = 'mbApiKey';
+
+// NOTE: On test "PropertiesService.getDocumentProperties()" cannot be used
+// and on running as Add-On after installed "testProperties" cannot be written(read only).
+let testProperties = {};
+
+function setProperty(key, value) {
+  const properties = PropertiesService.getDocumentProperties();
+  if (properties) {
+    properties.setProperty(key, value);
+  } else {
+    testProperties[key] = value;
+  }
+}
+
+function getProperty(key) {
+  const properties = PropertiesService.getDocumentProperties();
+  return properties ? properties.getProperty(key) : testProperties[key];
+}
+
+function setDeploymentId() {
   const ui = SpreadsheetApp.getUi();
+  const result = ui.prompt(
+    'Set Deployment ID',
+    'Please enter the deployment ID'
+    + ' (i.e. for "https://xxxxxxxxxxxxxxxxxxxxxxxxxx.multibaas.com",'
+    + ' only the "xxxxxxxxxxxxxxxxxxxxxxxxxx" part"):',
+    ui.ButtonSet.OK_CANCEL,
+  );
 
-  ui.createMenu('MultiBaas')
-    .addItem('Post to the blockchain', 'mbPost')
-    .addItem('Refresh current cell', 'mbRefresh')
-    .addToUi();
+  const button = result.getSelectedButton();
+  const text = result.getResponseText();
+  if (button === ui.Button.OK) {
+    setProperty(PROP_MB_DEPLOYMENT_ID, text);
+    ui.alert(`Deployment ID is ${text}.`);
+  }
 }
 
-/**
- * Menu 1
- */
-function mbRefresh() {
+function setApiKey() {
```

Comment:
Per my [comment](https://github.com/curvegrid/multibaas-for-google-sheets/pull/24#discussion_r471078612) in #24, it would be great if we could set Deployment and API key in a single dialogue.

Reply:
Same with https://github.com/curvegrid/multibaas-for-google-sheets/pull/24#discussion_r471420212